trace.go

```diff
@@ -75,13 +75,45 @@ func decodeTraceContext(raw string) (tc *TraceContext, err error) {
 	return
 }
 
-//EncodeTraceContext encodes the TraceContext into a string.
-//This is useful if you want to pass your trace context over an outgoing request
+// encodeTraceContext returns a concatenated string of all field values that exist in a trace context.
+func encodeTC(tc interface{}) string {
+	tcs := tc.(map[string]interface{})
+
+	m := map[string]string{
+		"PID": "",
+		"SID": "",
+		"TID": "",
+	}
+
+	for k, v := range tcs {
+		switch v.(type) {
+		case int:
+			m[k] = fmt.Sprintf("%v", tcs[k].(int))
+		case float64:
+			m[k] = fmt.Sprintf("%v", tcs[k].(float64))
+		case string:
+			m[k] = tcs[k].(string)
+		}
+	}
+
+	return fmt.Sprintf("%s=%v;%s=%v;%s=%v", pIDKey, m["PID"], sIDKey, m["SID"], tIDKey, m["TID"])
+}
+
+/*
+	fmt.Sprintf("%s=%v;%s=%v;%s=%v", pIDKey, tc[PID].(float64), sIDKey, tc[SID].(float64), tIDKey, tc[TID].(string))
+*/
+
+// EncodeTraceContext encodes the TraceContext into a string.
+func encodeTraceContext(tc *TraceContext) string {
+	return fmt.Sprintf("%s=%v;%s=%v;%s=%v", pIDKey, tc.PID, sIDKey, tc.SID, tIDKey, tc.TID)
+}
+
+// This is useful if you want to pass your trace context over an outgoing request or just need a string formatted trace context for any other purpose.
 func EncodeTraceContext(tc *TraceContext) string {
 	return fmt.Sprintf("%s=%v;%s=%v;%s=%v", pIDKey, tc.PID, sIDKey, tc.SID, tIDKey, tc.TID)
```

Comment:
simply the code by
return encodeTraceContext(tc)

Reply:
Sounds good